packages/portal/src/components/share/ShareSocialModal.vue

```diff
@@ -6,9 +6,11 @@
     hide-footer
     data-qa="share modal"
     title-tag="h2"
+    header-tag="div"
+    @show="$emit('show')"
   >
     <p>{{ $t('shareWhere') }}</p>
-    <div class="icon-wrapper pt-1">
+    <div class="icon-wrapper pt-1 mb-4">
       <ShareSocialButton
```

Comment:
unrelated, but this component should be called "shareSocialButtons" (plural) or the v-for statement moved out to here.

Reply:
I've renamed the component.